server/src/enum.ts

```diff
@@ -363,3 +363,11 @@ export enum ImmichWorker {
   API = 'api',
   MICROSERVICES = 'microservices',
 }
+
+export enum ImmichTelemetries {
```

Comment:
I would probably make this singular, none of the others are plural (`ImmichEnvironments`, `RouteKeys`, `MetadataKeys`, etc.)